packages/apps/transfer/src/utils/styles.ts

```diff
@@ -0,0 +1,32 @@
+import {IGridColMediaStyles, IGridColMediaType} from "@/types/Grid";
+
+const getPropInPercentage = (prop: number | string): string => {
+  const value = typeof prop === "string" ? parseInt(prop) : prop;
+  return `${(value / 12) * 100}%`
+};
+
+export const getColumnStyles = (prop: IGridColMediaType): IGridColMediaStyles => {
+  if (typeof prop === 'string') {
+    return {
+      width: getPropInPercentage(parseInt(prop)),
+    };
+  }
+
+  if (typeof prop === 'number') {
+    return {
+      width: getPropInPercentage(prop),
+    };
+  }
+
+  const { size, span, offset, pull, push, hidden } = prop;
+  const styles: IGridColMediaStyles = {};
+
+  if(size)  styles.width = getPropInPercentage(size).toString();
```

Comment:
needs to be done like this to make sure we don't have undefined properties in the result